src/screens/CreatePost/index.js

```diff
@@ -713,9 +689,11 @@
             setTypeUser={setTypeUser}
             isAnonymous={typeUser}
             anonUserInfo={anonUserInfo}
-            username={dataProfile.username ? dataProfile.username : 'Loading . . .'}
+            username={profile?.myProfile?.username}
             photo={
-              dataProfile.profile_pic_path ? {uri: dataProfile.profile_pic_path} : ProfileDefault
+              profile?.myProfile?.profile_pic_path
+                ? {uri: profile?.myProfile?.profile_pic_path}
+                : ProfileDefault
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `username` and `photo` props are now being derived from the `profile` state instead of `dataProfile`. Ensure that all instances of `dataProfile` have been replaced with `profile` to maintain consistency and avoid potential errors due to undefined variables.
```diff
+ username={profile?.myProfile?.username}
+ photo={
+   profile?.myProfile?.profile_pic_path
+     ? {uri: profile?.myProfile?.profile_pic_path}
+     : ProfileDefault
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->